internal/ignition/ignition_test.go

```diff
@@ -745,3 +746,50 @@ var _ = Describe("Lso manifests creation Test", func() {
 var _ = AfterEach(func() {
 	os.RemoveAll("manifests")
 })
+
```

Comment:
Add a test in the LSO section that will check that in case that OCS is enabled, g.checkLsoEnabled() should return true.